### PR TITLE
Fixes from previous PRs

### PR DESCRIPTION
**Pull request by BottleRocket-Colin**
Went back implemented comments from previous PRs.

### CODE REVIEW

File: domain/build.gradle.kts [hunk 1]
```diff
@@ -3,7 +3,8 @@ plugins {
     id("org.jetbrains.kotlin.jvm")
 }
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_7
-    targetCompatibility = JavaVersion.VERSION_1_7
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
 }
+
```
Comment (BottleRocket-Colin) [inline domain/build.gradle.kts:9]: Couldn't find a KotlinOptions to use here....  I guess it's something we don't have to worry about until we see an issue. In other projects it hasn't been needed.

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Bottle Rocket Android Architecture Guidelines
 
 ## OAuth consumer key creation and setup steps
 * Generate a Bitbucket OAuth consumer key at https://bitbucket.org/[your-bitbucket-username]/workspace/settings/api
-    * You must specify a callback url. The value isn't important, so go ahead and use https://www.bottlerocketstudios.com/
+    * You must specify a callback url, for auth code login to work you must: use https://www.bottlerocketstudios.com/
```
Comment (jaredstockton) [inline README.md]: @BottleRocket-Colin nit: Formatting. How about:

```
You must specify a callback url with the following value for auth code login to work: https://www.bottlerocketstudios.com/
```

```suggestion
    * You must specify a callback url with the following value for auth code login to work: https://www.bottlerocketstudios.com/
```

File: app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/BaseViewModel.kt [hunk 3]
```diff
@@ -40,23 +41,33 @@ abstract class BaseViewModel : ViewModel(), KoinComponent {
     fun launchIO(block: suspend CoroutineScope.() -> Unit): Job =
         viewModelScope.launch(dispatcherProvider.IO, block = block)
 
+    /**
+     * Utility function to switch coroutine context to Main.
+     *   Useful for making UI updates from IO
```
Comment (jaredstockton) [inline app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/BaseViewModel.kt]: @BottleRocket-Colin Copy looks looks good! Just a nit on the formatting. Either:

1. Combine both lines onto a single line OR
2. Adjust the spaces before Useful (second line) to match the spacing of Utility (first line)

File: app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/BaseViewModel.kt [hunk 4]
```diff
@@ -40,23 +41,33 @@ abstract class BaseViewModel : ViewModel(), KoinComponent {
...
+     */
+    suspend fun handleError(@StringRes messageId: Int) {
         runOnMain {
             toaster.toast(messageId)
         }
     }
 
-    suspend inline fun <T : Any> Status<T>.handlingErrors(messageID: Int , onSuccess: (T) -> Unit): Status<T> {
+    /**
+     * Used to apply default error when handling Status, and process a success block.
```
Comment (jaredstockton) [inline app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/BaseViewModel.kt]: @BottleRocket-Colin nit: Remove `,`

```suggestion
     * Used to apply default error when handling Status and process a success block.
```

**Review by jaredstockton — CHANGES_REQUESTED** [review]
Appreciate the quick turnaround! Looks good overall. Just mentioning a few minor nits. Once addressed, I'll approve 😄 . Would be nice if GitHub had an Approved with suggestions status like ADO does 🤷🏻

**Review by jaredstockton — APPROVED** [review]

**Comment by jaredstockton** [comment]
Nice job!

**Pull request merged**